src/theme/book.js

```diff
@@ -199,6 +199,18 @@ $( document ).ready(function() {
             }
         });
     });
+    
+    $("pre code").each(function(i, block){
+        var pre_block = $(this).parent();
+        if( !pre_block.hasClass('playpen') ) {
```

Comment:
I guess that we would prefer to have the button dock for each code/pre instead of specifically in the playgrounds. And only add additional buttons to (other than clipboard) to already existing button bar for rust specific snippets

Reply:
I'll wait for your comments then.